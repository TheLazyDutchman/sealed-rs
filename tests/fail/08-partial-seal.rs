use sealed::sealed;

#[sealed]
pub trait A {
	#[seal(callable)]
	fn b() {}
}

fn main() {}
