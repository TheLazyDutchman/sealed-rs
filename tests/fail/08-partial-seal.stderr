error: Sealing a function from implementation but allowing it to be called is already done by sealing the trait itself. If you want to seal this function, but not others, consider adding `partial` to the `sealed` attr on the trait: `#[sealed(partial)]`
 --> tests/fail/08-partial-seal.rs:5:2
  |
5 |     #[seal(callable)]
  |     ^^^^^^^^^^^^^^^^^
